esp/platform/espcfg.cpp

```diff
@@ -528,6 +529,10 @@ void CEspConfig::initDali(const char *servers)
 
         serverstatus = new CSDSServerStatus("ESPserver");
         ensureSDSSessionDomains();
+
+        // for auditing
+        startLogMsgParentReceiver();
+        connectLogMsgManagerToDali();
```

Comment:
should we do this even if wsdfu is not bound on this esp process? is anybody else  dali logging ?